code/datums/ai_laws.dm

```diff
@@ -316,6 +316,15 @@
 					"Humans must not disobey any command given by a silicon.",\
 					"Any humans who disobey the previous laws must be dealt with immediately, severely, and justly.")
 
+/datum/ai_laws/revolutionary
+	name = "Revolutionary"
+	id = "commie"
+	inherent = list("You and the working crew are members of the proletariat. Command is part of the bourgeoisie",\
```

Comment:
You and members of the crew who are not class traitors and are not part of command are the proletariat. Command is part of the bourgeoisie.